res/themes/light/css/_light.scss

```diff
@@ -168,9 +168,14 @@ $imagebody-giflabel-border: rgba(0, 0, 0, 0.2);
 $imagebody-giflabel-color: rgba(255, 255, 255, 1);
 // ********************
 
-$copy-button-url: "$(res)/img/feather-customised/clipboard.svg";
-$collapse-button-url: "$(res)/img/element-icons/minimise_collapse.svg";
-$expand-button-url: "$(res)/img/element-icons/maximise_expand.svg";
```

Comment:
Looks like a different PR that you merged in moved these lines further down in the file, so they are still present and should be removed from the "Icon URLs" section around line 300.

Reply:
Ah hmm, looks correct when viewing the full set of changes, so ignore me here.